lib/cli/handlers/status.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+
+const { logRed, logGreen, getComposeFileOrFlagsHandler } = require('./../methods');
+const { Polygon } = require('../../services/polygon');
+const { Terraform } = require('./../../../lib/services/terraform');
+const { prepareArgsForPolygonOps } = require('./common');
+
+async function status({ fileMode = false, __fileDirname = '', awsProfile, sshPublicKey: _sshPublicKey,
+    orbsAddress, orbsPrivateKey, region, nodeSize, nodeCount, publicIp,
+    cachePath: _catchPath, name, bootstrapUrl, backend = false, ephemeralStorage = false,
+    configPath: _configPath, chainVersion, ethereumEndpoint,
+    ethereumTopologyContractAddress, incomingSshCidrBlocks,
+    sslCertificatePath, sslPrivateKeyPath,
+}) {
+    const { cloud, keys } = prepareArgsForPolygonOps({
+        fileMode, __fileDirname, _configPath, _sshPublicKey, _catchPath,
+        chainVersion, ethereumTopologyContractAddress, sslCertificatePath, sslPrivateKeyPath,
+        orbsAddress, orbsPrivateKey, awsProfile, incomingSshCidrBlocks, ethereumEndpoint,
+        region, backend, nodeSize, nodeCount, bootstrapUrl, name, ephemeralStorage,
+    });
+
+    cloud.ip = publicIp;
+
+    const terraformAdapter = new Terraform();
+    terraformAdapter.setCachePath(path.join(process.cwd(), '_terraform'));
+
+    const polygon = new Polygon({ terraformAdapter });
+    polygon.setTerraformCachePath(cloud.cachePath);
+
+    const statusResult = await polygon.status({ cloud, keys });
+
+    if (statusResult.resourcesToAdd > 0 && statusResult.resourcesToAdd < 20) {
+        if (statusResult.resourcesToDestroy > 0 && statusResult.resourcesToDestroy < 20) {
+            logGreen('Your node is live on AWS');
+        }
+    } else if (statusResult.resourcesToAdd > 40) {
+        logRed('Your node has not been setup, please run polygon create to set it up on AWS');
```

Comment:
I'd like to suggest this output format:
1. If the manager node is provisioned (boyar instance) - `Your node is live on AWS`.  we then add: 
  - Number of misconfigured resources (if there are still resources to add or destroy)
  - If -v option was supplied, list all the arn's of resources to add (and to destroy?)
  - Running status of the AWS instance (manager)
1. if the manager node is not provisioned - 'Your node has not been created':  
  - Number of misconfigured resources (if there are still resources to destroy)
  - if -v option was supplied, list all the arn's of resources to destroy

What do you think?